fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java

```diff
@@ -226,6 +228,25 @@ private Void computeOlapScanNode(Operator node, ExpressionContext context, Table
         return visitOperator(node, context);
     }
 
+    @Override
+    public Void visitLogicalIcebergScan(LogicalIcebergScanOperator node, ExpressionContext context) {
+        return computeIcebergScanNode(node, context, node.getTable(), node.getColRefToColumnMetaMap());
+    }
+
+    @Override
+    public Void visitPhysicalIcebergScan(PhysicalIcebergScanOperator node, ExpressionContext context) {
+        return computeIcebergScanNode(node, context, node.getTable(), node.getColRefToColumnMetaMap());
+    }
+
+    private Void computeIcebergScanNode(Operator node, ExpressionContext context, Table table,
+                                   Map<ColumnRefOperator, Column> colRefToColumnMetaMap) {
```

Comment:
since the statistics has not been fully tested， i will submit another pr for iceberg statistics